src/thunderbots/software/CMakeLists.txt

```diff
@@ -63,6 +68,7 @@ catkin_package(
 
 # Specify additional locations of header files
 include_directories(
+	SYSTEM "${LIB_DIR}/Include"
```

Comment:
What does `${LIB_DIR}/Include` refer to?

Reply:
its incorrect, fixed it